DigitalLearningSolutions.Web/Services/PasswordResetService.cs

```diff
@@ -121,6 +123,17 @@ public void GenerateAndSendDelegateWelcomeEmail(int delegateId, string baseUrl)
             emailService.SendEmail(welcomeEmail);
         }
 
+        public Email GenerateDelegateWelcomeEmail(int delegateId, string baseUrl)
+        {
+            var delegateEntity = userService.GetDelegateById(delegateId)!;
+            var welcomeEmail = GenerateWelcomeEmail(

```

Comment:
It looks like this method generates email body text that includes spaces instead of line breaks. This is preexisting code in line 293 of Web\Services\PasswordResetService.cs. Please could we change this method so that it includes line breaks rather than spaces in the generated text?

Reply:
Replaced spaces with line breaks in email generated text.